### ISSUE TITLE
Compose drawer could squish down and become sticky with a ⬅️ button

### ISSUE THREAD

**Issue by Cassolotl** [tootsuite/mastodon#9868]
I saw a PR for making the entire compose drawer sticky, and I kinda like this idea, but I think on smaller screens that could be messy.

### Pitch

Here's my messy mock-up!

![photo_2019-01-19_22-08-48](https://user-images.githubusercontent.com/6607512/51432880-1b669f00-1c37-11e9-9294-3f5783d57c57.jpg)

![photo_2019-01-19_22-08-45](https://user-images.githubusercontent.com/6607512/51432881-228dad00-1c37-11e9-88b1-dac5aa3d3be2.jpg)

In words: When you press the ⬅️ button, the compose drawer and menu compress to the left and become sticky. The sticky menu bar thing has all the same icons on it, plus a "new toot" button and a ➡️ to pop the full compose drawer back out again.

The only thing that's missing is an avatar (to click to open your profile in the web UI) and the "edit profile" button. Maybe it could have a profile button too, or a tiny avatar, that you can click to open your profile in the farthest-right column? Then people could click "edit profile" from there.

### Motivation

The PR for making the compose drawer sticky drew my attention to the way it takes up so much space on my laptop screen and I don't use it that much. Being able to summon the new toot box when I need it, but squashing it away when I don't, would improve my screen situation a lot. Plus you'd be able to get to all the handy menu items without having to pop the compose drawer back out again.

**Comment by ronilaukkarinen** [comment]
Yes, this is really needed. The compose form should not be always in the way. I have moved it to last column on [Mastodonr Bird UI](https://github.com/ronilaukkarinen/mastodon-bird-ui/).
